### DISCUSSION TITLE
Looking for testers for v1.2-rc.0

### DISCUSSION THREAD

**Discussion by ukutaht** [plausible/analytics#605] [Anything Else]
Find the changelog and upgrade instructions here: https://github.com/plausible/analytics/releases/tag/v1.2-rc.0

If you do decide to try out the release candidate, please leave a message here. I will address any issues and if everything seems smooth I will create a proper release.

**Comment by Vigasaurus** [comment]
I can't comment on the extremely new (<4d) stuff yet (I'll be deploying the RC to my setup this weekend), but I've been consistently updating to the newest master, weekly, currently been running at 564d07ec61f55280d067316d0d78acc9900dfc55 since Monday - and all has run smoothly. 

Will let you know how the RC goes 👍

**Reply by ukutaht** [reply]
Thanks @Vigasaurus! Good to know.

Since then @oliver-kriska upgraded the Elixir version and I moved to the new `config/runtime.exs` option for configuration so there are some potentially risky changes in the RC that are not on your instance yet. So would be good to know about the RC as well.

**Reply by Vigasaurus** [reply]
Deployed and working well so far, even with it being a migration from a self-built intermediate image to the official image. 

One quick (tiny) thing I noticed off the bat though - the github logo svg (on the repo button) on the dashboard is always black, even in dark mode - just tacking on a `fill="currentcolor"` fixes it, though.

**Comment by winterrific** [comment]
Hello @ukutaht,

Upgrading to V1.2 RC.0 run smoothly on my docker based installation. No issues so far. I'll post it here, just when I'm getting any issues.

Best regards

**Comment by oliver-kriska** [comment]
New version of Elixir should not make problems. I use 1.11.2 couple of months without any issue. Sorry one issue, not sure if it's related to upgrade or my custom changes, some error with unsubscribe when email reports are generated.

I have custom changes because of our cloud provider and sendgrid. So I have couple of conflicts to resolve
https://github.com/payout-one/analytics/pull/11
<img width="184" alt="Screenshot 2021-01-15 at 19 55 47" src="https://user-images.githubusercontent.com/39122854/104767017-c7321f00-576b-11eb-87c4-8e681447008a.png">

I will release it and we will see.

**Reply by oliver-kriska** [reply]
released and working. We will see for couple of days.

**Comment by lizgw** [comment]
I upgraded a few days ago and it went smoothly, no problems so far. :+1:

**Comment by uhlhosting** [comment]
Upgrade was smooth!

UI Changes are snapy and fully working, love the new UX changes too.

For the time beying no issues to report. Are there any specific areas to test more in depth?

**Reply by ukutaht** [reply]
No, just a general 'click around make sure nothing breaks'. It's been out for a week now, a couple issues came but they were fixed. I think I'm releasing this in the stable channel very soon.

**Reply by uhlhosting** [reply]
It seems i catched a issue on ios safari beta 14.1

Here is printscreen.

![IMG_1103](https://user-images.githubusercontent.com/6324047/105346709-47caa280-5be6-11eb-8dbf-650399f578d2.PNG)

**Reply by ukutaht** [reply]
Thanks @uhlhosting. I don't see the same issue with my stable Safari on iOS. How did you install the beta?

**Reply by Vigasaurus** [reply]
I could totally see that happening on more than just Safari actually, looks like a timing issue on a component unmounting/the DOM child getting removed before willUnmount fires, probably pretty hard to reproduce. 
 https://github.com/plausible/analytics/blob/master/assets/js/dashboard/pinned-header-hoc.js#L22-L26 

Probably just needs a guard clause for existence of the DOM node ahead of observe and unobserve calls

**Comment by uhlhosting** [comment]
I have no idea what you say there this is very reproducible error on my setup.
